tests/testutils.py

```diff
@@ -35,56 +32,10 @@
 
 REQ_PATH = b"/_matrix/push/v1/notify"
 
-USE_POSTGRES = environ.get("TEST_USE_POSTGRES", False)
-# the dbname we will connect to in order to create the base database.
-POSTGRES_DBNAME_FOR_INITIAL_CREATE = "postgres"
-POSTGRES_USER = environ.get("TEST_POSTGRES_USER", None)
-POSTGRES_PASSWORD = environ.get("TEST_POSTGRES_PASSWORD", None)
-POSTGRES_HOST = environ.get("TEST_POSTGRES_HOST", None)
-
 
 class TestCase(unittest.TestCase):
     def config_setup(self, config):
-        self.dbname = "_sygnal_%s" % (time_ns())
-        if USE_POSTGRES:
-            config["database"] = {
-                "name": "psycopg2",
-                "args": {
-                    "user": POSTGRES_USER,
-                    "password": POSTGRES_PASSWORD,
-                    "database": self.dbname,
-                    "host": POSTGRES_HOST,
-                },
-            }
-        else:
-            config["database"] = {"name": "sqlite3", "args": {"dbfile": ":memory:"}}
-
-    def _set_up_database(self, dbname):
-        conn = psycopg2.connect(
-            database=POSTGRES_DBNAME_FOR_INITIAL_CREATE,
-            user=POSTGRES_USER,
-            password=POSTGRES_PASSWORD,
-            host=POSTGRES_HOST,
-        )
-        conn.autocommit = True
-        cur = conn.cursor()
-        cur.execute("DROP DATABASE IF EXISTS %s;" % (dbname,))
-        cur.execute("CREATE DATABASE %s;" % (dbname,))
-        cur.close()
-        conn.close()
-
-    def _tear_down_database(self, dbname):
-        conn = psycopg2.connect(
-            database=POSTGRES_DBNAME_FOR_INITIAL_CREATE,
-            user=POSTGRES_USER,
-            password=POSTGRES_PASSWORD,
-            host=POSTGRES_HOST,
-        )
-        conn.autocommit = True
-        cur = conn.cursor()
-        cur.execute("DROP DATABASE %s;" % (dbname,))
-        cur.close()
-        conn.close()
+        super
```

Comment:
typo? Unfinished line?

Reply:
probably just remove this function I would've thought

Reply:
It was a lack of confidence on my part-I removed it and it blew up a bunch of tests so I got scared and left it. I just did a bit of investigation and fixed everything now so it's removed and the tests are now fine-hopefully it's correct!